src/logging/logger.go

```diff
@@ -53,7 +54,7 @@ func configure(config Config) *zerolog.Logger {
 }
 
 func newRollingFile(config Config) io.Writer {
-	if err := os.MkdirAll(config.Directory, 0744); err != nil {
+	if err := os.MkdirAll(config.Directory, 0740); err != nil {
```

Comment:
@vishnusomank Why do re provide execute permission for the user ?
It could be 640 right ?

Reply:
yes 640 would be enough. I changed it into 740 from 744 because to pass lint. I can change it into 640 if required @seswarrajan

Reply:
@seswarrajan i've tested it by changing permission into 0640. But some of the functionalities were failing with error `permission denied`. can we keep it 740 itself?

Reply:
740 is fine